CHANGELOG.md

```diff
@@ -49,6 +49,7 @@
 * [CHANGE] Increased default `-<prefix>.redis.timeout` from `100ms` to `500ms`. #3301
 * [CHANGE] `cortex_alertmanager_config_invalid` has been removed in favor of `cortex_alertmanager_config_last_reload_successful`. #3289
 * [CHANGE] Query-frontend: POST requests whose body size exceeds 10MiB will be rejected. The max body size can be customised via `-frontend.max-body-size`. #3276
+* [FEATURE] Cassandra: Add `-cassandra.table-operations-consistency` flag that allows a separate consistency level to be set for keyspace/table operations to Cassandra. #3402
```

Comment:
I guess strictly speaking this is also a `CHANGE`, as table operations that had been run with a consistency of `QUORUM` before are now run as `ALL`

Reply:
Makes sense. I think it might be best to make the default `QUORUM` so this isn't a breaking change for anyone currently using this.